bridgesync/processor.go

```diff
@@ -211,6 +215,45 @@ func (p *processor) GetClaims(
 	return claims, nil
 }
 
+func (p *processor) GetBridgesPaged(
+	ctx context.Context, pageNumber, pageSize uint32, depositCount uint64,
+) ([]*Bridge, int, error) {
+	tx, err := p.db.BeginTx(ctx, &sql.TxOptions{ReadOnly: true})
+	if err != nil {
+		return nil, 0, err
+	}
+	defer func() {
+		if err := tx.Rollback(); err != nil {
+			log.Warnf("error rolling back tx: %v", err)
+		}
+	}()
+	orderBy := "deposit_count"
+	order := "DESC"
+	whereClause := ""
+	count, err := p.GetTotalNumberOfRecords("bridge")
+	if err != nil {
+		return nil, 0, err
+	}
+	if depositCount > 0 {
+		whereClause = fmt.Sprintf("WHERE deposit_count = %d", depositCount)
+		pageNumber = 1
+		pageSize = 1
+	}
+	offset := (pageNumber - 1) * pageSize
+	rows, err := p.queryPaged(tx, offset, pageSize, "bridge", orderBy, order, whereClause)
```

Comment:
1. Do we need to call `rows.Close` in deferred execution?

2. Use the const for the bridge table name
```suggestion
	rows, err := p.queryPaged(tx, offset, pageSize, bridgeTableName, orderBy, order, whereClause)
```